### PR TITLE
Testrunnertk- Cypress sans saucectl new

### PR DESCRIPTION
**Pull request by walkerlj0**
Thank you for Contributing to Sauce School. Please add in the following to your PR:

Updated modeules 1.02 through 1.04 of the Cypress on Sauce course. Two major changes
- Removed mentions of 'saucectl new' and changed in setup workflow instructions to download an example suite
- Moved mentions of 'Docker Mode' away from the top/ main instructions and emphasized sauce mode as the main way to run tests
- Removed mentions of running with `saucectl --test-env`

### Types of changes

- [ ] Platform Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Major Documentation Update (New modules, courses, or major re-organizations)
- [x] Minor Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-school/blob/master/CONTRIBUTING.MD) document.
- [ ] For feature updates: I have updated the documentation [changelog.md](https://github.com/saucelabs/sauce-school/blob/master/changelog.md) accordingly.
- [ ] I have added tests to cover my changes.
- [x] I ran tests before merging & [tests passed](https://github.com/saucelabs/sauce-school/blob/master/TESTING.md).

### CODE REVIEW

File: codelabs/Module1_TestRunner.md [hunk 1]
```diff
@@ -156,52 +139,40 @@ First, anywhere on your machine install the Saucectl tool globally, using this c
...
-```
-saucectl configure
-```
-
-This command prompts you to manually enter your credentials if it cannot detect any environment variables, and will generate a `credentials.yml` file in a .sauce directory in your home folder.
-
 ### Cypress Test Code
 
-You want to create a `saucectl new` project at the same level as the **cypress** directory that contains your tests.
+You want to fork and clone or download a .zip copy of the [example project](https://docs.saucelabs.com/testrunner-toolkit/installation), then place your tests in the **cypress** directory that contains your tests.
```
Comment (alexplischke) [inline codelabs/Module1_TestRunner.md:144]:
```suggestion
You want to fork, clone or download a .zip copy of the [example project](https://docs.saucelabs.com/testrunner-toolkit/installation), then place your tests in the **cypress** directory that contains your tests.
```

File: codelabs/Module1_TestRunner.md [hunk 2]
```diff
@@ -156,52 +139,40 @@ First, anywhere on your machine install the Saucectl tool globally, using this c
...
 
-<img src="assets/TRT1.03A.png" alt="Create project directory" width="400"/>
+### Set Sauce Username and Access Key
 
-To start the toolkit and create your testrunner project, run the command `saucectl new`, then choose the Cypress framework.  Choose the region nearest to you, and you should have the Saucectl client up and running.
+Watch [this video](https://www.youtube.com/watch?v=3K1Eu0eTha8&t=12s) to see how to set up your Sauce username and access key as environment variables on your machine, or use [the instructions here to set them up on Windows](https://docs.google.com/document/d/1Cb27j6hgau5JHmAxGHPihd3V4Og3autPCei82_m1Ae8/edit?usp=sharing).
 
-<img src="assets/TRT1.03B.png" alt="Choose a framework with Sauce C T L" width="800"/>
+Testrunner Toolkit will detect your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`, but you can run the following optional command to edit these:
```
Comment (alexplischke) [inline codelabs/Module1_TestRunner.md:161]: `saucectl` :D not `Testrunner Toolkit`.

File: codelabs/Module1_TestRunner.md [hunk 3]
```diff
@@ -128,26 +128,9 @@ _More detailed instructions are below_
...
-#### Note
-
-Negative
-: Typically, one would need to import a Docker image, however since you installed Saucectl, the image has been specified in the config file for Saucectl and imported for you.
-
-
 ### Download and Install Saucectl
 
-Next you need to download and install the Sauce Control Command Line Interface (CLI) that you will use to run Testrunner Toolkit. This is a part of Testrunner Toolkit that allows you to set a configuration location & update the file in your local directory.  There are several options (*Insert link) for installing it, and in this tutorial we will use npm, which means you need to have [NodeJS installed on your machine](https://nodejs.org/en/download/).
+Next you need to download and install the Sauce Control Command Line Interface (CLI) that you will use to run Testrunner Toolkit. This is a part of Testrunner Toolkit that allows you to set a configuration location & update the file in your local directory.  There are several options (https://docs.saucelabs.com/testrunner-toolkit/installation) for installing it, and in this tutorial we will use npm, which means you need to have [NodeJS installed on your machine](https://nodejs.org/en/download/).
```
[the remark this answers is not in the document]
Comment (walkerlj0) [reply codelabs/Module1_TestRunner.md:133]: is it 'saucectl'? Will likely do a search and replace for entire course (modules 1 & 2) once I am given official word via Rachel and Kunal

File: codelabs/Module1_TestRunner.md (hunk 2 above)
Comment (walkerlj0) [reply codelabs/Module1_TestRunner.md:161]: will change all later

**Comment by walkerlj0** [comment]
Approved one change, will change all mentions of 'TestRunner Toolkit' at a later time

**Pull request merged**